examples/simple_walk_forward/README.md

```diff
@@ -19,7 +19,7 @@ ros2 launch spot_driver spot_driver.launch.py
 
 5.  Run the example:
 ```bash
-ros2 run simple_walk_forward walk_forward
+ros2 run simple_walk_forward walk_forward --robot <spot_name>
```

Comment:
Since the example now works with or without a robot name that should be explained here.

Reply:
Should also be done in the readme for simple_arm_motion